salt/apache/init.sls

```diff
@@ -85,6 +85,20 @@ disable site 000-default.conf:
   file.absent:
     - name: /var/www/html/index.html
 
+/etc/apache2/conf-available/zz-customization.conf:
+  file.managed:
+    - contents: |
+       ServerTokens Prod
+       ServerSignature Off
+
```

Comment:
I figure follow same requirement/dependency pattern as other `/etc/apache2/` files, unless the alternative in this PR is an improvement?

```suggestion
    - require:
      - pkg: apache2
    - watch_in:
      - module: apache2-reload

```